src/editor/Core/ConfigurationValues.re

```diff
@@ -57,10 +57,3 @@ let default = {
   filesExclude: ["node_modules", "_esy"],
   zenModeHideTabs: true,
 };
-
-let getBundledConfigPath = () => {
```

Comment:
This wasn't being used anyways.

Reply:
Awesome! Thanks for removing it - one less thing to think about 😄